src/context/UserDataContext.js

```diff
@@ -58,6 +60,23 @@ const UserDataProvider = ({ children }) => {
 					console.log("error", err);
 				}
 			})();
+		auth.isAuth &&
+			(async () => {
+				setLoading(true);
+				try {
+					const res = await getAddressListService(auth.token);
+					console.log("addresses", res);
```

Comment:
Remove consoles, (removing consoles after functionality works is almost all we forgot.)